portsyncd/portsyncd.cpp

```diff
@@ -92,11 +92,14 @@ int main(int argc, char **argv)
         netlink.dumpRequest(RTM_GETLINK);
         cout << "Listen to link messages..." << endl;
 
-        if (!port_config_file.empty())
+        if (!handlePortConfigFromConfigDB(p, cfgDb, warm))
         {
-            handlePortConfigFile(p, port_config_file, warm);
-        } else {
-            handlePortConfigFromConfigDB(p, cfgDb, warm);
+            // if port config is missing in ConfigDB
+            // attempt to use port_config.ini
+            if (!port_config_file.empty())
+            {
+                handlePortConfigFile(p, port_config_file, warm);
+            }
```

Comment:
should we "throw" if port_config_file.empty() is true in this else branch?

Reply:
I tried to keep the old logic as it was before